components/Context/DdfInfo.js

```diff
@@ -24,9 +28,158 @@ export default class DdfInfo extends React.Component {
+          //filters always start off as checked
+          processedIssues[category].filters[label] = true;
+        }
+      })
+    });
+
+    this.setState({issues: processedIssues, categories})
+ }
+
+ getContractAddress(issue, comments){
+  //TODO Change from owner to status' username once jose implements the end point
+  comments.filter(comment => comment.author_association === "OWNER");
+  let contractAddress = -1;
+  let match = comments[0].body.match(ethereumRegex());
+  if(match && match.length > 0){
+    contractAddress = match[0];
+  }
+  return {
+    ...issue,
+    contractAddress,
+    value: "$100"
+  };
+ }
+
+async getCommentsOfIssue(repoName, number){
+  return new Promise(async (resolve, reject) => {
+    axios.get(`https://api.github.com/repos/csmartinsfct/${repoName}/issues/${number}/comments?access_token=573dbede2e9766fd4d41610d15a8b82371fd522a`)
+      .then(function (response) {
+        // handle success
+       resolve(response.data);
+      })
+      .catch(function (error) {
+        // handle error
+        console.log(error);
+      })
+    })
+ }
+
+ getCategory(repoName){
+  if(repoName.indexOf("tech") !== -1 || repoName.indexOf("website") !== -1)
+    return "Development";
+  else if(repoName.indexOf("design") !== -1){
+    return "Design";
+  }
+  else if(repoName.indexOf("marketing") !== -1)
+    return "Marketing";
+  //TODO Remove after implementing jose's end point - just because my repos don't have these new names yet :x
+  else if(repoName === "MyBit-Go")
+    return "Development";
+  else
+    return "Design";
+}
+
+ async getIssues(){
+  try{
+    let issuesTmp = await Promise.all(Repos.map(async (repoName) =>
+      axios.get(`https://api.github.com/repos/csmartinsfct/${repoName}/issues?access_token=573dbede2e9766fd4d41610d15a8b82371fd522a`)
+        .then(response => {
+          const data = response.data;
+          return data.map(issue => {
+            let indexOf = issue.html_url.indexOf('/issues');
+            const repoUrl = issue.html_url.substring(0, indexOf + 1);
+            const labels = issue.labels.map(label => label.name);
+            const repoName = issue.html_url.split('/')[4];
+            const category = this.getCategory(repoName);
+            return {
+              issueUrl: issue.html_url,
+              title: issue.title,
+              createdAt: issue.created_at,
+              number: issue.number,
+              state: issue.state,
+              repoUrl: repoUrl,
+              repoName: repoName,
+              labels,
+              category,
+            }
+          })
+        })
+        .catch(function (error) {
+          throw error;
+        })
+    ))
+
+    //This won't be the case in production since we'll use a different endpoint that returns all issues
+    //I'm testing pulling issues from two different repos atm so need to merge the arrays
+    let issues = issuesTmp[0].concat(issuesTmp[1]);
```

Comment:
Why though?

Reply:
I'm making two requests to pull data from two different repos, hence I get two arrays back. With your end point is just one request for the whole org.

Reply:
I think we are making ton of assumptions about our backend. That's fine, but be aware we might set up GraphQL, so just think about how you are getting the data for that component only, and not the logic you have to do over it.